ui/lib/kubernetes/addon/components/page/roles.js

```diff
@@ -32,7 +32,6 @@ export default class RolesPageComponent extends Component {
     try {
       const message = `Successfully deleted role ${model.name}`;
       await model.destroyRecord();
-      this.args.roles.removeObject(model);
```

Comment:
Destroying record automatically removes it from the store, so no need to update `this.args.roles` list directly